demo/md-styles.html

```diff
@@ -0,0 +1,787 @@
+<link rel="import" href="../polymer/polymer-element.html">
```

Comment:
highlightjs stylesheet modified to have the slot prefix

I'm wondering if `:host marked-element` is needed in the selector?